pages/connect-account.tsx

```diff
@@ -56,8 +56,10 @@ export default function ConnectAccount() {
   function redirectToProfile() {
     const lastNetworkVisited = new WinStorage(`lastNetworkVisited`, 0, 'localStorage');
 
+    const toNetworks = state.Service?.active?.network ? "/networks" : "/setup"
```

Comment:
can't we simply _grab the current page_ and pass that instead?

Reply:
is the current page the connect-account, in this case would it be the previous page?